src/Comps/search/recentSearches/RecentSearches.tsx

```diff
@@ -0,0 +1,59 @@
+import React, {
+  Dispatch,
+  MouseEventHandler,
+  SetStateAction,
+  useEffect,
+  useState,
+} from "react";
+import { Container, Row, Title, ClearButton, Search } from "./style";
+import {} from "../../../styles/layouts";
+import {} from "../../../styles/typography";
+import { DeleteIcon } from "../../../Icons";
+
+type ISearch = {
+  value: string[];
```

Comment:
Avoid using the words items/values/data/content and other generic naming as much as possible